### PR TITLE
LPD-17784 LPD-17785

### PR DESCRIPTION
**Pull request by istvansajtos**

### CODE REVIEW

**Comment by istvansajtos** [comment]
ci:test:relevant

**Comment by istvansajtos** [comment]
ci:test:security

**Comment by istvansajtos** [comment]
ci:test:sf

**Comment by istvansajtos** [comment]
ci:test:relevant

**Comment by istvansajtos** [comment]
ci:test:security

**Comment by stian-sigvartsen** [comment]
@istvansajtos this looks really good I think.

I noticed a few `System.out.println` statements in there which you need to remove before sending the follow up PR with SF fixes.

**Comment by stian-sigvartsen** [comment]
@istvansajtos  I mentioned earlier that when submitting the Create Account form there is a redirect to the LOGIN portlet instead of current portlet. It looks like there is a simple fix for this in `LoginUtil.getLoginURL` . Can you have a look?

**Comment by istvansajtos** [comment]
ci:test:sf

**Comment by istvansajtos** [comment]
ci:test:relevant

**Comment by istvansajtos** [comment]
ci:test:security

**Comment by istvansajtos** [comment]
ci:test:sf

**Comment by istvansajtos** [comment]
ci:test:relevant

**Comment by istvansajtos** [comment]
ci:test:security

**Comment by istvansajtos** [comment]
Hi @stian-sigvartsen ,

I've applied the following changes/updates:
- Removed the System.out.println statements - thank you for catching them!
- Regarding the Create Account form redirecting to Login portlet (you mentioned [here](https://github.com/liferay-appsec/liferay-portal/pull/1673#issuecomment-2004316643)), I added a feature flag check to CreateAccountMVCActionCommand [here](https://github.com/liferay-appsec/liferay-portal/pull/1673/commits/5a88529dd73ce8800e791bcc12f77cfdfce1d754#diff-9eccc9de4dbf60128cb970096037cfcd36a305b6a27c618c25b06e68cfd82230R422-R424).
- Reworded commit messages to pass SF
- Extracted indent in configuration.jsp from the application of the feature flag for transparency. See new commit [LPD-17784 SF - indent](https://github.com/liferay-appsec/liferay-portal/pull/1673/commits/08143a6f7bb1020b5e5a511821da53bfd1984024)
- Switched from <liferay-ui:tags> to <clay:tags> to pass SF, see new commit [LPD-17784 SF - Use <clay:tabs> tag instead of <liferay-ui:tabs>](https://github.com/liferay-appsec/liferay-portal/pull/1673/commits/6a49154477b5bb6d9708e04e993f0457a9670d9b)

To sum up, apart from the [redirect fix](https://github.com/liferay-appsec/liferay-portal/pull/1673/commits/5a88529dd73ce8800e791bcc12f77cfdfce1d754#diff-9eccc9de4dbf60128cb970096037cfcd36a305b6a27c618c25b06e68cfd82230R422-R424) everything is basically SF related.

**Comment by istvansajtos** [comment]
@stian-sigvartsen , all the the test failures seems to be unrelated.

**Comment by stian-sigvartsen** [comment]
Looks great Istvan. I found a behavioural regression with the redirect fix so I've sent you a PR to this branch/PR. See istvansajtos/liferay-portal/pull/85

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
ci:test:sf

**Review by stian-sigvartsen — APPROVED** [review]

**Comment by stian-sigvartsen** [comment]
Everything looks good, manually forwarded to brianchandotcom/liferay-portal/pull/148449
